prices/README.md

```diff
@@ -0,0 +1,107 @@
+# Price component
+
+## Price design specs
+
+You can find the design specs on [www.decathlon.design](https://www.decathlon.design)
+
+## Usage with Vitamin Theme
+
+By inheriting the Vitamin theme in your app, you can use the `Vitamin*Price` component.
+
+```kotlin
+implementation("com.decathlon.vitamin:prices:<version>")
+```
+
+```xml
+<com.decathlon.vitamin.prices.VitaminPriceDefaultSmall 
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content" 
+    android:text="150 €" />
+```
+
+With the same usage as above, you have multiple price types available:
+
+| Button type             | Kotlin Class                         | Attribute style                     |
+|-------------------------|--------------------------------------|-------------------------------------|
+| Default Small           | `VitaminPriceDefaultSmall`           | `?attr/priceDefaultSmall`           |
+| Default Medium          | `VitaminPriceDefaultMedium`          | `?attr/priceDefaultMedium`          |
+| Default Large           | `VitaminPriceDefaultLarge`           | `?attr/priceDefaultLarge`           |
+| Accent Large            | `VitaminPriceAccentSmall`            | `?attr/priceAccentSmall`            |
+| Accent Medium           | `VitaminPriceAccentSmall`            | `?attr/priceAccentMedium`           |
+| Accent Large            | `VitaminPriceAccentSmall`            | `?attr/priceAccentLarge`            |
+| Alert Small             | `VitaminPriceAlertSmall`             | `?attr/priceAlertSmall`             |
+| Alert Medium            | `VitaminPriceAlertMedium`            | `?attr/priceAlertMedium`            |
+| Alert Large             | `VitaminPriceAlertLarge`             | `?attr/priceAlertLarge`             |
+| Strikethrough Small     | `VitaminPriceStrikethroughSmall`     | `?attr/priceStrikethroughSmall`     |
+| Strikethrough Medium    | `VitaminPriceStrikethroughMedium`    | `?attr/priceStrikethroughMedium`    |
+| Strikethrough Large     | `VitaminPriceStrikethroughLarge`     | `?attr/priceStrikethroughMedium`    |
```

Comment:
`?attr/priceStrikethroughMedium` should not be `?attr/priceStrikethroughLarge` ?